commands/serve/lib/fixtures.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Manages loading and delivering fixtures.
+ *
+ * Fixtures are transpiled and included when Nebula serve is starting. Fixtures in the specified `--fixturePath` with the extension `.fix.js` are included.
+ */
+window.fixtures = {
```

Comment:
since this is put on the window scope, maybe give it a little bit more specific name to avoid clashes... `serveFixtures` for instance.